pkg/service/providerindex/providerindex.go

```diff
@@ -50,10 +50,10 @@ func New(providerStore types.ProviderStore, findClient ipnifind.Finder, publishe
 
 // Find should do the following
 //  1. Read from the IPNI Storage cache to get a list of providers
-//     a. if there are no records in the cache, attempt to ready claims from legacy systems -- Dynamo tables & content
+//     a. if there are no records in the cache, query IPNI, filtering out any non-content claims metadata
+//     b. if no records are found in IPNI, attempt to ready claims from legacy systems -- Dynamo tables & content
```

Comment:
I think ready is a typo.
```suggestion
//     b. if no records are found in IPNI, attempt to read claims from legacy systems -- Dynamo tables & content
```